src/includes/getting-started-install/apple.mdx

```diff
@@ -9,4 +9,6 @@ target 'YourApp' do
 end
 ```
 
-Afterwards run `pod install`.
+Then run `pod install`.
+
+We also support alternate [installation methods](install/).
```

Comment:
Whatever we agree about the comment on the apple.ios.mdx would go here too

Reply:
see previous